third_party/terraform/utils/storage_transfer_helpers.go

```diff
@@ -0,0 +1,312 @@
+package google
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/schema"
+	"google.golang.org/api/storagetransfer/v1"
+	"time"
+)
+
+func expandDates(dates []interface{}) []*storagetransfer.Date {
```

Comment:
nit: expand/flatten funcs should go in the main resource file below CRUD/import methods (at the very bottom generally)

Reply:
Moved to bottom and deleted the helpers file.